grazer/pom.xml

```diff
@@ -215,6 +215,19 @@
 				</exclusion>
 			</exclusions>
 		</dependency>
+		
+		
+		
+		
+		<!-- Hibernate and Dependent jars -->
+		<!-- Hibernate core -->
+		<dependency>
+			<groupId>org.hibernate</groupId>
+			<artifactId>hibernate-core</artifactId>
+			<version>5.2.10.Final</version>
+		</dependency>
+
+		
```

Comment:
I had accidentally written this into this ticket, and didn't delete it. I moved this into EPGR-3